pypeit/bitmask.py

```diff
@@ -351,12 +351,12 @@ def turn_on(self, value, flag):
 
         out = value | (1 << self.bits[_flag[0]])
         if len(_flag) == 1:
-            return out
+            return out.astype(value.dtype)
```

Comment:
Worth @kbwestfall checking this.

Reply:
This is necessary due to a discrepancy in the result types of bitwise operations between numpy arrays of type ``np.int32`` and ``int`` on Windows vs macOS/Linux
macOS <img width="682" alt="numpy_int32" src="https://user-images.githubusercontent.com/5816532/93643107-29446880-f9b4-11ea-9d47-e8922e329de6.png">
Windows ![numpy_intc](https://user-images.githubusercontent.com/5816532/93643121-2fd2e000-f9b4-11ea-8e4a-f49236a439c9.PNG)
I'll propagate this to other bitwise operations in ``bitmask.py``